CryptoAnalysis/src/main/java/crypto/typestate/FiniteStateMachineToTypestateChangeFunction.java

```diff
@@ -77,7 +78,8 @@ public FiniteStateMachineToTypestateChangeFunction(SootBasedStateMachineGraph fs
 
 	@Override
 	protected State initialState() {
-		throw new RuntimeException("Should never be called!");
+		LOGGER.error("Should never be called!");
+		return null;
```

Comment:
If it really should never be called then throw an exception here. Is there some sort of OperationNotSupportedException in Java? Because this is not really a CryptoAnalysisException...